### DISCUSSION TITLE
Installation and Usage Guide

### DISCUSSION THREAD

**Discussion by dr-code** [mkdocs-publisher/mkdocs-publisher#56] [General]
Hi there, first of all I’ve been looking for a materials with mkdocs set up that connects with obsidian without losing the backlinks and graphs, so thank you for this.

that being said, I’m a total noob when it comes to this stuff, I just want to be able to publish my notes so that I can read them on the fly, with your set up as it as what I’m looking for.

I’ll be honest I’m not sure where to exactly start with your installation.

so far I have created a new Obsidian vault with docs, _template, and .obsidian folder. I have a git repository, that has .gitignore, the docs folder that I opened with Obsidian. I’m not really sure where to go from there to get everything up and running. I assume the mkdocs.yml file gets generated and I think it’s from the mkdocs new *project name*

I apologize for the very basic question that likely requires a more elaborate answer. Appreciate any help! Thanks

**Comment by mkusz** [comment]
Hi @dr-code,
As for now, I know my project documentation is lacking in that area, and it will improve over time, but for now I can help you here or on other private channels (you can find me on Obsidian discord for example). If you want to share your git repo with me, I can see what you have and what next you should do. If not, the first thing for you should be to create `mkdocs.yml` file and edit it. Probably the best option at the beginning would be to copy and edit that file from this project repo. Just have in mind that you have to change name and URL. If you want a web page like this repo official documentation. You will need to create a GitHub Pages and setup some automations, but let's start with the basics, and when you will be ready for next step, just let me know here.

**Comment by dr-code** [comment]
Hello

I have attached my entire folder in a zip format below so that you can see what I have done. I can delete it from my post once you have had a chance to take a look at it. 

Here is the error I got when I committed and uploaded the deploy_pages action. 
<img width="1412" alt="image" src="https://github.com/mkusz/mkdocs-publisher/assets/8603561/ead3528c-c523-4aa5-b20a-40cbf9a01781">

Thanks for you assistance, I can start from scratch as well since I didn't copy any of my notes over to this project yet. 

[Project.zip](https://github.com/mkusz/mkdocs-publisher/files/12983660/Project.zip)

**Reply by mkusz** [reply]
I have answered to yours issue. I'm closing this thread since it's a copy of an issue.
